pipelines/reconfigure-contributor.yml

```diff
@@ -0,0 +1,31 @@
+resource_types:
+- name: concourse-pipeline
+  type: registry-image
+  source:
+    repository: concourse/concourse-pipeline-resource
+
```

Comment:
I imagine you copy-pasted, but I can't help but noticed this is unused!
```suggestion
```

Reply:
Interesting I wonder why the pipeline validation didn't capture this unused resource type